src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  todosData: Todo[];
+  filter: 'All' | 'Active' | 'Completed';
+  setFilter: (filter: 'All' | 'Active' | 'Completed') => void;
+  onClearClick: () => void;
+  isDeleting: boolean;
+};
+
+export const Footer: React.FC<Props> = ({
+  todosData,
+  filter,
+  setFilter,
+  onClearClick,
+  isDeleting,
+}) => {
+  return (
+    <>
+      {/* Hide the footer if there are no todos */}
+      {todosData.length > 0 && (
+        <footer className="todoapp__footer" data-cy="Footer">
+          <span className="todo-count" data-cy="TodosCounter">
+            {todosData.filter(todo => !todo.completed).length} items left
+          </span>
+
+          {/* Active link should have the 'selected' class */}
+          <nav className="filter" data-cy="Filter">
+            <a
+              href="#/"
+              className={classNames('filter__link', {
+                selected: filter === 'All',
+              })}
+              data-cy="FilterLinkAll"
+              onClick={() => setFilter('All')}
+            >
+              All
+            </a>
+
+            <a
+              href="#/active"
+              className={classNames('filter__link', {
+                selected: filter === 'Active',
+              })}
+              data-cy="FilterLinkActive"
+              onClick={() => setFilter('Active')}
+            >
+              Active
+            </a>
+
+            <a
+              href="#/completed"
+              className={classNames('filter__link', {
+                selected: filter === 'Completed',
+              })}
+              data-cy="FilterLinkCompleted"
+              onClick={() => setFilter('Completed')}
+            >
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method